Training5/manageapp/src/redux/action/ActionNames.js

```diff
@@ -0,0 +1,5 @@
+export const ACTION_NAMES = {
+    LOGIN_USER: "LOGIN_USER",
```

Comment:
@Huynhkimphat 
Each api call should have 3 actions: includes success & error action